pyproject.toml

```
@@ -46,9 +46,9 @@ dynamic = ["version"]
 # TODO: List the packages and pin them to a version that the tests in your project depend on
 [project.optional-dependencies]
 test = [
-    "pytest>=5.4",
+    "pytest>=",
     "pytest-cov",
-    "pytest-asyncio>=0.17",
+    "pytest-asyncio>=",
```

There is no need to have the comparison operators when there's no version specified 
```suggestion
    "pytest-asyncio,
```

---

My thinking was if we completely remove the comparison operators, a new contributor like myself would be unsure which packages need version specification.
Follow-up question: I just checked [pypi](https://pypi.org/project/pytest-cov/), and I realize I never specified a min version for pytest-cov. Is it necessary?

---

> would be unsure which packages need version specification.

The general recommendation is to [*always* pin the versions](https://pip.pypa.io/en/stable/topics/repeatable-installs/#pinning-the-package-versions) for all dependencies. This is because, when you don't pin, the latest available version will be installed, and if that comes with breaking changes, your package might get broken.

> Follow-up question: I just checked [pypi](https://pypi.org/project/pytest-cov/), and I realize I never specified a min version for pytest-cov. Is it necessary?

You are refering to pinning the package [here](https://github.com/jupyterhub/jupyterhub-pytest-plugin/blob/main/pyproject.toml#L39), right?
Yes, we should pin that too.

---

Alright. I will pin the version for the pytest-cov package as well.